### ISSUE TITLE
new errors

### ISSUE THREAD

**Issue by Unknown7001** [9-FS/nhentai_archivist#43]
I got these error messages today:

Thank you

```
WARN  Saving hentai metadata page 1 / <unknown> in database failed with: Invalid image type: "w" at line 1 column 2884
WARN  Saving hentai metadata page 2 / <unknown> in database failed with: Invalid image type: "w" at line 1 column 3025


WARN  acquired connection, but time to acquire exceeded slow threshold aquired_after_secs=2.4139999 slow_acquire_threshold_secs=2.0

WARN  Downloading hentai metadata page 1.953 / 4.629 from "<unknown>" failed with: error decoding response body

WARN  Downloading hentai metadata page 1.958 / 4.629 from "https://nhentai.net/api/galleries/search?query=language:%22English%22&page=1958" failed with: error sending request for url (https://nhentai.net/api/galleries/search?query=language:%22English%22&page=1958)

WARN  slow statement: execution time exceeded alert threshold summary="PRAGMA foreign_keys = OFF; …" db.statement="\n\nPRAGMA foreign_keys = OFF;\nBEGIN\nTRANSACTION;\nINSERT\n  OR REPLACE INTO Hentai (\n    id,\n    cover_type,\n    media_id,\n    num_favorites,\n    num_pages,\n    page_types,\n    scanlator,\n    title_english,\n    title_japanese,\n    title_pretty,\n    upload_date\n  )\nVALUES\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),\n  (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);\nINSERT\n  OR REPLACE INTO Tag (id, name, type, url)\nVALUES\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?),\n  (?, ?, ?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nDELETE FROM\n  Hentai_Tag\nWHERE\n  hentai_id = ?;\nINSERT INTO\n  Hentai_Tag (hentai_id, tag_id)\nVALUES\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?),\n  (?, ?);\nCOMMIT;\nPRAGMA foreign_keys = ON;\n" rows_affected=1252 rows_returned=0 elapsed=1.7577541s elapsed_secs=1.7577541 slow_threshold=1s```

**Comment by 9-FS** [comment]
What nhentai ID has resulted in this error? I need that to reproduce the error and verify my fix.

**Comment by 9-FS** [comment]
@Irodzuita What nhentai ID has resulted in this error? I need that to verify my fix.

**Comment by 9-FS** [comment]
Fixed in version 3.5.1..
